. Procfile

```diff
@@ -0,0 +1 @@
+web: node src/app.js
```

Comment:
In production, its better if we run the es5 version of our app in `/dist` forlder